source/conf.py

```diff
@@ -49,10 +57,15 @@
     'q2doc.qiime1',
     'q2doc.checkpoint',
     'q2doc.command_block',
-    'q2doc.plugin_directory',
     'q2doc.external_links',
 ]
 
+
+generate_plugin_rst()
+
+root = pathlib.Path(__file__).parent
```

Comment:
Please move this up with the import path section, that way you can reuse the same logic in both places.